### DISCUSSION TITLE
求助大佬，关于使用ws协议  反向代理实现内网穿透

### DISCUSSION THREAD

**Discussion by JimmyCaro** [v2fly/v2ray-core#1970] [Q&A]
如题，我想用ws协议来实现内网穿透，无奈Nginx配置卡脖子

现有windows A -  vps B - windows C 三台设备

想将A的请求流量交给Nginx转发给C
C的返回流量通过Nginx返回给A
如此就可以实现双通道流量加密

跪舔大佬给个整套的配置代码，全网找遍了没找到合适的。

**Comment by crarm** [comment]
我也有这样的需求，找到一个教程 [使用v2ray进行反向代理/内网穿透v](http://www.hellosong.top/2021/07/11/%E4%BD%BF%E7%94%A8v2ray%E8%BF%9B%E8%A1%8C%E5%8F%8D%E5%90%91%E4%BB%A3%E7%90%86-%E5%86%85%E7%BD%91%E7%A9%BF%E9%80%8Fv/)
还没有搞成功

**Reply by wmxhw** [reply]
感谢指路

**Comment by lhbdhr** [comment]
为啥不用专门的异地组网软件？例如zerotier
